tweb/src/main/frontend/src/js/vue/components/ProjectCard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+	<div class="project-container">
+		<div class="project-icon">
+			<img v-bind:src="project.logoLocation">
+		</div>
+		<a v-bind:href="project.projectLink">
+			<div class="project-request-details" :style="{ 'background': project.overviewColor }">
+				<div class="project-request-name"><h3>{{ project.name }}</h3></div>
+				<div class="project-request-description"><p>{{ project.description }}</p></div>
+				<hr />
```

Comment:
If you add a container around the hr with the requestCountContainer. You can align-self: flex-end (this will make sure the hr is on the same line for each element. 
In extreme cases it can look like this.
<img width="244" alt="screen shot 2018-06-26 at 17 24 15" src="https://user-images.githubusercontent.com/1853096/41922620-cb85dc86-7965-11e8-9062-24b797772e5c.png">